build/scripts/entrypoint-volume.sh

```diff
@@ -72,5 +72,11 @@ if [ -f /tmp/che/secret/ca.crt ]; then
   export NODE_EXTRA_CA_CERTS=/tmp/che/secret/ca.crt
 fi
 
+if [ -z "$NODEJS_FOR_VSCODE_DIR" ]; then
```

Comment:
maybe we need better variable name here...

Reply:
maybe `VSCODE_NODEJS_RUNTIME_DIR` ?